Sources/Network/Connection/ProtocolConnectionFactory.swift

```diff
@@ -17,9 +17,7 @@ protocol ProtocolConnectionFactory {
 
     func makeConsensusService(
         config: AttestedConnectionConfig<ConsensusUrl>,
-        targetQueue: DispatchQueue?,
```

Comment:
here too

Reply:
in PR #188, I restored all the rng code that was in place prior to starting idempotence.